code/addons/highlight/src/preview.ts

```diff
@@ -5,55 +5,104 @@ import { global } from '@storybook/global';
 
 import { addons } from 'storybook/preview-api';
 
-import { HIGHLIGHT, HIGHLIGHT_STYLE_ID, RESET_HIGHLIGHT } from './constants';
+import { HIGHLIGHT, RESET_HIGHLIGHT, SCROLL_INTO_VIEW } from './constants';
 
 const { document } = global;
 
-type OutlineStyle = 'dotted' | 'dashed' | 'solid' | 'double';
-
-const highlightStyle = (color = '#FF4785', style: OutlineStyle = 'dashed') => `
-  outline: 2px ${style} ${color};
-  outline-offset: 2px;
-  box-shadow: 0 0 0 6px rgba(255,255,255,0.6);
-`;
-
-interface HighlightInfo {
-  /** HTML selector of the element */
+interface HighlightOptions {
+  /** HTML selectors of the elements */
   elements: string[];
-  color: string;
-  style: OutlineStyle;
+  /** Color of the outline */
+  color?: string;
```

Comment:
I'm assuming these docs need to be updated? (we can do it in a different PR)
https://storybook.js.org/docs/essentials/highlight

Reply:
The existing docs are still valid. I added a couple new options for the "jump to element" behavior but I plan to change that with the highlight-overlay PR, so I'd rather not document those for now.